### ISSUE TITLE
How to divide training set and validation set?

### ISSUE THREAD

**Issue by chouxianyu** [njvisionpower/Safety-Helmet-Wearing-Dataset#43]
Could you tell us how did you divide the training set and validation set?
Is it random?
If so, how is the proportion allocated？
9(train):1(val)？
You can reply in Chinese！

**Comment by qwe3208620** [comment]
写脚本自动划分。。。

**Comment by chouxianyu** [comment]
@qwe3208620 
哈哈，我知道应该是自动划分的。
我的问题是：划分规则是什么？这个划分是随机划分的吗？

**Comment by cym3223** [comment]
> @qwe3208620
> 哈哈，我知道应该是自动划分的。
> 我的问题是：划分规则是什么？这个划分是随机划分的吗？

作者的数据集目录VOC2028/ImageSets/Main/下，有划分的图片序号
